src/RokuDeploy.ts

```diff
@@ -731,6 +716,70 @@ export class RokuDeploy {
         return this.doPostRequest(deleteOptions);
     }
 
+    /**
+     * Gets a screenshot from the device. A side-loaded channel must be running or an error will be thrown.
+     */
+    public async takeScreenshot(options?: RokuDeployOptions & { screenshotPath?: string }) {
```

Comment:
Let's create a specific interface for this function that includes the options we care about (rootDir, host, password). We only use like 3 of the RokuDeployOptions.

Reply:
Sure.